test/functional/services/combo_box.ts

```diff
@@ -65,21 +73,22 @@ export function ComboBoxProvider({ getService, getPageObjects }: FtrProviderCont
       await this.openOptionsList(comboBoxElement);
 
       if (value !== undefined) {
-        const options = await find.allByCssSelector(
+        const selectOptions = await find.allByCssSelector(
           `.euiFilterSelectItem[title^="${value.toString().trim()}"]`,
```

Comment:
I guess this isn't a change in this PR, but I don't understand why we're only looking for the case where the element's title attribute starts with value.  I would think we would want to find the element who's title equals value?  Maybe this was an issue with data-test-subj's with spaces in them?  If that issue is fixed maybe we could improve this here for exact match?

Reply:
It is used to match drop-down option that starts with provided text. I remember testing related PR and may confirm we need it for some tests